Port `v` and `V` to RzShell

Rot127 opened the pull request:
<!-- Filling this template is mandatory -->

**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

<!-- Explain the **details** for making this change. Is a new feature implemented? What existing problem does the pull request solve? How does the pull request solve these issues? Please provide enough information so that others can review your pull request. -->

This brings some changes:

- Visual commands passed to `V`, must be given as normal argument (space between command and string).
- Changes "load layout" `v [layout-name]` to `vl [layout-name]`, to have a hard distinction between `v` and `vl`.
- Change "store layout" `v= [layout-name]` tp `vs [layout-name]`. Because it is more descriptive and matches `vl` a little better.
- Drops the vi command for two reasons:
  - vi (open file in editor) is not at all related to the visual mode. Hence shouldn't be in the same group.
  - (Opinionated) Users can just open the file in another terminal window. Why go the extra way over Rizin?

**Test plan**

<!-- What steps should the reviewer take to test your pull request? Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots/videos. This is your time to re-check that everything works and that you covered all the edge cases -->

All green

**Closing issues**

<!-- put "closes #XXXX" in your comment to auto-close the issue that your PR fixes (if any). -->

...

Rot127 committed:
Print log messages if inserting of description fails.

Rot127 committed:
Port V command to RzShell

Rot127 committed:
Port v (panels) commands to RzShell and drop vi.

Drops the vi command for two reasons:

- vi (open file in editor) is not at all related to the visual mode.
Hence shouldn't be in the same group.
- (Opinionated) Users can just open the file in another
terminal window. Why go the extra way over Rizin?

Replace old command names with something more descriptive.

- v [name] -> vl name
- v= -> vs

Rot127 committed:
Fix return values

Rot127 committed:
Add missing space before V command.

XVilka commented on librz/core/cmd_descs/cmd_interactive.yaml:
```diff
@@ -0,0 +1,58 @@
+          - text: ""
+            arg_str: "Press: <TAB>"
+            comment: Toggles different configurations of the current printing mode.
+      - name: Examples
```
Add more information what the argument means - a sequence of the pressed keys in the visual mode. Also, it's better to have a few examples for those cases.

XVilka replied:
Also, this is not full help. Either implement a full help or make `V?` corresponding to the one press of `?` in the `V` mode, and `V??` corresponding to two presses of `?` in the `V` mode.

Rot127 replied:
Then I go with `VH`/`VHH` for showing the help of the `V` mode. `V?` should be really just for the command.
But I'd like to have more details available to the user before they enter the V mode.
I remember I got confused my first time. It was the "How do I exit vim moment".

XVilka replied:
Ok to have `VH` for the full help, but still examples section and explanation what the argument exactly is needed in the `V?` and `V??` output.

XVilka reviewed (changes requested):
In my practice, commands like `Vp`, `Vpp`, and `VP` are common, maybe add some aliases for those particular cases?

Rot127 committed:
Split up V and v files to allow for definitino of Vp and Vpp commands

Rot127 committed:
Restructure help for visual mode.

Add VH and VHH for key list in the visual mode.
Remove examples, but add a more detailed description about the key-sequence parameter.

wargio reviewed (approved):
i'm ok with these.

XVilka commented:
@Rot127 please rebase one more time to resolve the conflict

Rot127 committed:
Fix rebase mistakes
